config/publisher.php

```diff
@@ -19,4 +23,11 @@
         'nova-api*',
         'nova-vendor*',
     ],
+    'conflicts' => [
+        'schema' => ConflictSchema::class,
+        'queue' => 'sync',
+        'listener' => DetectSchemaConflicts::class,
+        'job' => ResolveSchemaConflicts::class,
+        'resolver' => KeyResolver::class,
```

Comment:
This is used in case someone uses non-standard primary keys on any models.

Reply:
I do wonder if since the `KeyResolver` is exclusively used in the `DetectSchemaConflicts` listener, if its `fromTable()` method could just live there, since we can already override that. Though i do recognize that having them separate is a good application of SOLID. I'm mostly thinking about cognitive complexity there, tbh; Mostly a nit.